docs/RuntimeAttributes.md

```diff
@@ -369,6 +371,14 @@ runtime {
 }
 ```
 
+#### How to Setup
+
+Configure your Google network to use "Private Google Access".  This will allow your VM's to access Google Services including Google Container Registry, as well as Dockerhub.
```

Comment:
```suggestion
Configure your Google network to use "Private Google Access". This will allow your VMs to access Google Services including Google Container Registry, as well as Dockerhub images.
```